src/components/teaser/teaser.scss

```diff
@@ -57,17 +61,15 @@
   flex-flow: var(--sbb-teaser-flex-direction) nowrap;
   align-items: var(--sbb-teaser-align-items);
   gap: var(--sbb-teaser-gap);
-  height: var(--sbb-teaser-container-height, auto);
 }
 
 ::slotted([slot='image']) {
   will-change: transform;
   display: block;
-  object-fit: cover;
-  width: 100%;
+  width: var(--sbb-teaser-image-width, 300px);
```

Comment:
we would like to discuss the default behavior once more with @mcilurzo

Reply:
pushed a change in order to have the correct hover effect